src/ccstruct/publictypes.h

```diff
@@ -161,22 +161,22 @@ enum TextlineOrder {
  * so that the inequality test macros below work.
 */
 enum PageSegMode {
-  PSM_OSD_ONLY,       ///< Orientation and script detection only.
-  PSM_AUTO_OSD,       ///< Automatic page segmentation with orientation and
+  PSM_OSD_ONLY,       ///< 0 - Orientation and script detection only.
```

Comment:
```suggestion
  PSM_OSD_ONLY = 0,       ///< Orientation and script detection only.
```

Reply:
I suggest to add the values to the definition, not in the comment.

Reply:
Done via 259d5af6b1d9d193053ec579e42a2c6a5fcb9b63